src/app/helpers/filters/filters.ts

```diff
@@ -60,5 +61,68 @@ export function filterModel<T extends AbstractModel, U extends AbstractModel>(
     },
   };
 
-  return filterAnd(currentFilter, additionalFilter);
+  return filterAnd<T>(currentFilter, additionalFilter);
+}
+
+/**
+ * Adds a new filter to an existing filter that filters by several model ids
+ *
+ * @param key The key identifier of the model, most likely the type
+ * @example key = "projects"
+ * @param model The model to filter by
+ * @param currentFilter (optional) A filter that model condition will be added to. If no filter is provided, a new filter will be created
+ * @returns A new filter with the model conditions added
+ */
+export function filterModelIds<T extends AbstractModel>(
+  key: string,
+  ids: Id[] | Ids,
+  currentFilter: InnerFilter<T> = {}
+): InnerFilter<Writeable<T>> {
+  if (!isInstantiated(ids)) {
+    return currentFilter;
+  }
+
+  const additionalFilter: InnerFilter = {
+    [`${key}.id`]: {
+      in: ids,
+    },
+  };
+
+  return filterAnd<T>(currentFilter, additionalFilter);
+}
+
+export function propertyFilter<T extends AbstractModel, K extends keyof T>(
+  key: K,
+  value: T[K],
+  currentFilter: InnerFilter<T> = {}
+): InnerFilter<Writeable<T>> {
+  if (!isInstantiated(value)) {
+    return currentFilter;
+  }
+
+  const additionalFilter: InnerFilter = {
+    [key]: {
+      contains: value,
```

Comment:
name doesn't match function. should be `contains`?

also the `contains` operation only works on textual fields so the typing is not restrictive enough